sdk/spring/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 ## Release History
 
-### 5.3.0-beta.1 (Unreleased)
-
-#### Features Added
```

Comment:
Should we mention that Spring Data Cosmos supporting Spring Data 3.x in the features added?

Reply:
The PR is https://github.com/Azure/azure-sdk-for-java/pull/34878.